src/neo-vm/ExecutionEngine.cs

```diff
@@ -419,32 +421,277 @@ private bool ExecuteInstruction()
                         if (!context.EvaluationStack.Reverse((int)n.ToBigInteger())) return false;
                         break;
                     }
-                case OpCode.DUPFROMALTSTACKBOTTOM:
+                case OpCode.ISNULL:
                     {
-                        Push(context.AltStack.Peek(-1));
+                        if (!TryPop(out StackItem x)) return false;
+                        Push(x.IsNull);
                         break;
                     }
-                case OpCode.DUPFROMALTSTACK:
+
+                //Slot
+                case OpCode.INITSSLOT:
                     {
-                        Push(context.AltStack.Peek());
+                        if (instruction.TokenU8 == 0) break;
+                        if (context.StaticFields != null) return false;
+                        context.StaticFields = new Slot(instruction.TokenU8, ReferenceCounter);
                         break;
                     }
-                case OpCode.TOALTSTACK:
+                case OpCode.INITSLOT:
                     {
-                        if (!TryPop(out StackItem x)) return false;
-                        context.AltStack.Push(x);
+                        if (instruction.TokenU8 > 0)
+                        {
+                            if (context.LocalVariables != null) return false;
```

Comment:
Maybe we should put this line before the condition. INITSLOT WITH 0 should FAULt

Reply:
I mean, should fault if previously was called with 1, for example

Reply:
Done.